tests/unit/main_html5/wrapper-api-tests.js

```diff
@@ -374,6 +381,13 @@ describe('main_html5 wrapper tests', function () {
   });
 
   it('should remove closed captions if language is null', function(){
```

Comment:
I do see that you have a few pretty good CC tests in wrapper-event-tests, but, just in case, I would like to bring to your attention CC tests that I do in wrapper-api-tests for bitmovin plugin.  I do understand that some of these tests are irrelevant to main_html5 plugin, but I would recommend to review them - I think some of them are relevant to this version of wrapper-api-tests and you may want to implement them here as well.  You can check out pbw-4563 from video-plugins-internal to see the actual code, but I will quote headers of theses tests here so it may help you out without going to this code:

  it('should set VTT closed captions', function(){
  it('should set DFXP closed captions', function(){
  it('\'setClosedCaptions\' should show closed captions if params argument is not provided', function(){
  it('\'setClosedCaptions\' should show closed captions if closedCaptions argument is not provided', function(){
  it('\'setClosedCaptions\' should hide closed captions if mode in params argument is set to \'hidden\'', function(){
  it('\'setClosedCaptions\' should disable closed captions if \'language\' argument is empty or null', function(){
  it('should disable closed captions if setClosedCaptionsMode is \'disabled\'', function(){
  it('should show closed captions if setClosedCaptionsMode is \'showing\'', function(){
  it('should hide closed captions if setClosedCaptionsMode is \'hidden\'', function(){
  it('should remove closed captions if language is null', function(){
  it('should remove closed captions on setClosedCaptions if mode is not \'showing\'', function(){
  it('should set player\'s trackId to URL of in-manifest closed captions', function(){
  it('for \'en\' live closed captions player\'s trackId is \'01\' and subtitle label is \'en live\'' , function(){
  it('should overwrite player\'s previously existing subtitles with live closed captions of the same language', function(){
  it('should create new closed captions ids for newly added languages', function(){